ctapipe/calib/camera/mc.py

```diff
@@ -0,0 +1,578 @@
+"""
+Integrate sample-mode data (traces) Functions
+and
+Convert the integral pixel ADC count to photo-electrons
+"""
+
+import sys
+import numpy as np
+from pyhessio import *
+from ctapipe import io
+from astropy import units as u
+
+__all__ = [
+    'set_integration_correction',
+    'pixel_integration_mc',
+    'full_integration_mc',
+    'simple_integration_mc',
+    'global_peak_integration_mc',
+    'local_peak_integration_mc',
+    'nb_peak_integration_mc',
+    'calibrate_amplitude_mc',
+]
+
+CALIB_SCALE = 0.92
+
+"""
+
+The function in this module are the same to their corresponding ones in
+- read_hess.c
+- reconstruct.c
+in hessioxxx software package, just in some caes the suffix "_mc" has
+been added here to the original name function.
+
+Note: Input MC version = prod2. For future MC versions the calibration
+function might be different for each camera type.
+It has not been tested so far.
+
+In general the integration functions corresponds one to one in name and
+functionality with those in hessioxxx package.
+The same the interpolation of the pulse shape and the adc2pe conversion.
+
+"""
+
+
+def qpol(x, np, yval):
+    ix = round(x)
```

Comment:
This function fails for me because `round()` doesn't exist. I think you need to do:
```python
from numpy import round
```
at the top